src/ResourceManager/KeyVault/AzureRM.KeyVault.Netcore.psd1

```diff
@@ -0,0 +1,162 @@
+﻿#
+# Module manifest for module 'PSGet_AzureRM.KeyVault'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 4/3/2017
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+# RootModule = ''
+
+# Version number of this module.
+ModuleVersion = '0.9.6'
+
+# Supported PSEditions
+CompatiblePSEditions = 'Core'
+
+# ID used to uniquely identify this module
+GUID = 'cd188042-f215-4657-adfe-c17ae28cf730'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = 'Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = '[PowerShell .Net Core] Microsoft Azure PowerShell - KeyVault service cmdlets for Azure Resource Manager'
+
+# Minimum version of the Windows PowerShell engine required by this module
+PowerShellVersion = '5.1'
+
+# Name of the Windows PowerShell host required by this module
+# PowerShellHostName = ''
+
+# Minimum version of the Windows PowerShell host required by this module
+# PowerShellHostVersion = ''
+
+# Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# DotNetFrameworkVersion = '4.0'
+
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# CLRVersion = '4.0'
+
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
+
+# Modules that must be imported into the global environment prior to importing this module
+RequiredModules = @(@{ModuleName = 'AzureRM.Profile.Netcore'; ModuleVersion = '0.9.6'; })
+
+# Assemblies that must be loaded prior to importing this module
+# RequiredAssemblies = @()
```

Comment:
All dependencies not included in AzureRM.Profile.NetCore.psd1 will need to be listed here in RequiredAssemblies (as I obtusely noted below).  This generally includes the management client assembly